core/utils/poly-look/src/poly-look.js

```diff
@@ -1,7 +1,5 @@
-import "./styles.css";
-import "./themes.css";
-import "./colors.css";
-
+import { cssBundler } from "./css";
+cssBundler();
```

Comment:
Just so the linter doesn't complain - Could also just import all CSS files here

Reply:
I think I'd prefer that :D

Reply:
OK, I see now. You can also simply put a comment to turn off that warning here.

Reply:
What would that be?

Reply:
It depends on the error, but it's something like // eslint-disable whatever, like this one: You need to look at the name of the error, and just put it there.

```
features/polyExplorer-v1/src/components/dataViz/companyBubbles.jsx:/* eslint-disable no-case-declarations */
```

Reply:
Agreed, if you want to bundle all CSS imports into one file (which I wouldn't, but still :D), disabling the eslint warning would be nicer.

Reply:
I do suspect there might be a nicer way to bundle multiple CSS files into a single one via Rollup, this whole detour via JS stops making sense if we the CSS files aren't per-component, but global :D Maybe not something for this PR though.

Reply:
I think we do want css per component and this is just the base. Do we really want to have two ways to bundle css then?